.travis.yml

```diff
@@ -29,19 +29,14 @@ install:
 - ccache --max-size=5G
 
 before_script:
-# patch build system for the build type being processed
-- "[ ! -f tests/travis.patch ] || patch -p1 --no-backup-if-mismatch < tests/travis.patch"
-- "[ ! -f tests/travis-${BUILD_TYPE}.patch ] || patch -p1 --no-backup-if-mismatch < tests/travis-${BUILD_TYPE}.patch"
-
-# set up ccache
-- sg lxd -c '/snap/bin/lxc profile set default environment.PATH "/usr/lib/ccache:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin"'
-- sg lxd -c '/snap/bin/lxc profile device add default ccache disk source=${HOME}/.ccache/ path=/root/.ccache'
-- ccache --zero-stats
+# determine build identifier
+- |
+  [[ -n "${TRAVIS_PULL_REQUEST}" && "${TRAVIS_PULL_REQUEST_BRANCH}" != release/* ]] && export MULTIPASS_BUILD_ID="pr.${TRAVIS_PULL_REQUEST}"
```

Comment:
Yeah too many dots to my liking. Maybe `0.9.0-dev300.pr123+g20cca791`?
Or `0.9.0-dev.300.pr123+g20cca791`, since "300" isn't really part of the "dev" bit.

Reply:
Yeah, sounds reasonable.